projects/online_sequence_classification/clustering/clustering_interface.py

```diff
@@ -0,0 +1,275 @@
+import heapq
+import numpy as np
+
+from abc import ABCMeta, abstractmethod
+
+
+
+class Point(object):
```

Comment:
Actually, no I shouldn't update that part. I forgot that named tuples are immutable, and I am mutating Point.value (the cluster centroid are points and their value is updated dynamically)